packages/salesforcedx-vscode-visualforce/src/extension.ts

```diff
@@ -106,7 +106,7 @@ export async function activate(context: ExtensionContext) {
 
   let disposable = client.start();
   toDispose.push(disposable);
-  client.onReady().then(() => {
+  await client.onReady().then(() => {
```

Comment:
I'm not sure if this is the right way to do this. This would make it sequential, I think.

Reply:
I've updated this code to use a catch that will send an error to telemetry instead of an await.
```
.catch(err => {
      // Handled by clients
      telemetryService.sendExtensionActivationEvent(err);
    });
```